### ISSUE TITLE
multiple gpus_cannot access second gpu

### ISSUE THREAD

**Issue by pr4deepr** [lmb-freiburg/Unet-Segmentation#56]
Hi 
   I have a question about using multiple gpus with FIJI U-Net plugin.
I use Ubuntu 18.10 and have two GPUs, 
- NVIDIA Quadro P6000 (~24GB)
- Geforce RTX 2080 (8GB)
The display is connected to the RTX card and I would like to use the Quadro for U-Net segmentation and training, especially on 3D datasets, where VRAM is higher. 
When I run nvidia-smi, I get:
```
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 440.31       Driver Version: 440.31       CUDA Version: 10.2     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  Quadro P6000        Off  | 00000000:23:00.0 Off |                  Off |
| 26%   25C    P8     8W / 250W |      2MiB / 24449MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   1  GeForce RTX 2080    Off  | 00000000:2D:00.0  On |                  N/A |
|  0%   38C    P8    20W / 245W |    345MiB /  7974MiB |     14%      Default |
+-------------------------------+----------------------+----------------------+
```
When I run U-Net segmentation via the FIJI plugin and select all GPUs or GPU0, it runs on the RTX card. If I try GPU1 or any other GPUs after that, it says:
```
Model/Weight check failed:
See log for further details
```
I can access both the GPUs via ImageJ using the [CLIJ plugin](https://clij.github.io/). Both GPUs appear in the CLIJ plugin dropbox and I can run operations using them. 

Thanks
Pradeep

**Comment by ThorstenFalk** [comment]
Can you post the end of the log? It would be interesting to know which cuda error occurred. It looks like our caffe_unet build does not support the P6000 (I never had one for testing). Maybe you must recompile caffe_unet and enable higher compute capabilities. Although I'd assume backward compatibility, I remember we had similar problems with a TitanV in the past.

The setting "All available" in the GPU selection will only perform processing on the first available GPU (so basically equivalent to GPU0), thus you don't get the error in this mode.

**Comment by pr4deepr** [comment]
Thanks for the reply. I recompiled caffe again enabling higher computer capabilities (61). I did get an error saying gcc 8 and above are not supported as I had gcc 9, so I used update-alrernatives method to install and enable gcc 8. The compiling worked and I tried running the U-Net plugin on the P6000 and it seemed to work, but my computer was freezing and I had to force restart..  I've been using P6000 as my second GPU till now and quick image processing tasks
